bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -64,16 +64,24 @@ limitations under the License.
         <groupId>org.checkerframework</groupId>
         <artifactId>checker-qual</artifactId>
         <version>3.31.0</version>
+        <scope>provided</scope>
```

Comment:
Please add a comment why we're marking them as provided

Reply:
done